src/index.ts

```diff
@@ -1,92 +1,166 @@
 class School {
     // implement 'add area', 'remove area', 'add lecturer', and 'remove lecturer' methods
+    _areas: string[] = [];
+    _lecturers: string[] = []; // Name, surname, position, company, experience, courses, contacts
```

Comment:
_lecturers_ is not a list of strings. It's the list of objects with properties like name, surname, etc.